pkg/api/validation/validation.go

```diff
@@ -1039,6 +1039,14 @@ func ValidateService(service *api.Service) errs.ValidationErrorList {
 	if len(service.Spec.Ports) == 0 {
 		allErrs = append(allErrs, errs.NewFieldRequired("spec.ports"))
 	}
+	if service.Spec.Type == api.ServiceTypeLoadBalancer {
```

Comment:
That will just fail when we try to open the node port, and we'll see an error.  The trouble with the firewall is that it will silently expose the kubelet to the external internet.